src/components/Menu.jsx

```diff
@@ -1,29 +1,75 @@
 import React, { useState, useEffect } from "react";
-import { Segment, List } from "semantic-ui-react";
+import { Segment, Grid, Menu, Container } from "semantic-ui-react";
 import MenuItem from "./MenuItem";
 import axios from "axios";
 
-const Menu = () => {
+const MenuPage = () => {
   const [menuItems, setMenuItems] = useState([]);
+  const [activeCategory, setActiveCategory] = useState("starters");
+
   useEffect(() => {
     axios.get("https://slowfood.heroku.com/api/products").then((response) => {
       setMenuItems(response.data.products);
     });
   }, []);
+  useEffect(() => {
+    axios.get("https://slowfood.heroku.com/api/products").then((response) => {
+      setMenuItems(response.data.products);
+    });
+  }, [activeCategory]);
```

Comment:
I don't think you need 2 useEffects here

Reply:
I have removed one and It's still works.

Reply:
Yeah... The question is what `it` is that still works. ;) See my comment below.